config/core/configmaps/deployment.yaml

```diff
@@ -76,6 +76,19 @@ data:
     # If omitted, no value is specified and the system default is used.
     queue-sidecar-ephemeral-storage-limit: "1024Mi"
 
+    # Sets tokens associated with specific audiences for the queue proxy's QPOptions
+    #
+    # For example, to add the `service-x` audience:
+    #    queue-sidecar-tokens: "service-x"
+    # Also supports a list of audiences, for example:
+    #    queue-sidecar-tokens: "service-x,service-y"
+    # If omitted, or empty, no tokens are created
+    queue-sidecar-tokens: ""
+
+    # Sets rootCA for the queue proxy's QPOptions
```

Comment:
We do not use RootCA in QP (today) - since QP does not approach any service as a TLS client. 
It is correct that if we ever need to approach any TLS Server, the same RootCA will apply. I will try aim to improve the current comment